doc/changelog.txt

```diff
@@ -190,7 +190,7 @@ Fixes:
  - Fix using Set<Any>RulesParam with an incorrect type leaving that param set with a value of 0.
  - Fix COB SetMaxReloadTime receiving a value 10% smaller than it should
  - Fix builders not placing nanoframes from their maximum range
-
+-  Fix AllyTeam data retrievals not consistently using TeamHandler as source
```

Comment:
i'd leave this out because there aren't any actual behaviour effects. gamedevs are completely unaffected and the change is essentially internal cleanup

Reply:
Remove!